demo/www/lib/ngCordova/dist/ng-cordova.js

```diff
@@ -5007,7 +5011,7 @@ angular.module("ngCordova.plugins.oauth", ["ngCordova.plugins.oauthUtility"])
             var deferred = $q.defer();
             if(window.cordova) {
                 var cordovaMetadata = cordova.require("cordova/plugin_list").metadata;
-                if(cordovaMetadata.hasOwnProperty("org.apache.cordova.inappbrowser") === true) {
+                if(cordovaMetadata.hasOwnProperty("org.apache.cordova.inappbrowser") === true || cordovaMetadata.hasOwnProperty("cordova-plugin-inappbrowser") === true || cordovaMetadata.hasOwnProperty("cordova-plugin-inappbrowser") === true) {
```

Comment:
You are making twice the same test here:
```javascript
cordovaMetadata.hasOwnProperty("cordova-plugin-inappbrowser") === true || cordovaMetadata.hasOwnProperty("cordova-plugin-inappbrowser") === true
```
Remove one, I think we could put this string in a variable, it's better than repeat it, or maybe in a array and just test if the value is in it, cause then if someday the name change it again we can just add the new name to the variable, what do you think about @JoshAntBrown ?